morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Well that suggests that the two procedures should be re-using the same underlying but not exposed function. 
I think the main argument is that a boolean typically indicates a function is trying to do two different things. In this case the two different things are related, but I still think the use case of `.popup(x, y, true)` isn't clear. (Unless you define `showSubMenus = true`, and you use that... [also x and y are placeholders since those are bad names for anything other than coordinates, pretty much.])